### PR TITLE
[MIG] migrate fleet to new API

### PR DESCRIPTION
**Pull request by lla-odoo**
https://www.odoo.com/web#id=10075&view_type=form&model=project.task&menu_id=3942&action=327

### CODE REVIEW

File: addons/fleet/__openerp__.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
-    'name' : 'Fleet Management',
-    'version' : '0.1',
-    'author' : 'OpenERP S.A.',
+    'name': 'Fleet Management',
+    'version': '1.0',
+    'author': 'Odoo S.A.',
```
Comment (WHPODOO) [inline addons/fleet/__openerp__.py]: No need to write ***author***

File: addons/fleet/models/fleet.py [hunk 2]
```diff
@@ -0,0 +1,627 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import defaultdict
+import datetime
+from dateutil.relativedelta import relativedelta
+
+from odoo import tools, _, models, api, fields
```
Comment (WHPODOO) [inline addons/fleet/models/fleet.py]: Can you arrange in alphabetic order :question: 
```
from odoo import api, fields, models, tools, _
```

File: addons/fleet/views/fleet_views.xml [hunk 3]
```diff
@@ -1002,6 +1000,84 @@
...
+              sequence="1"
+              groups="group_fleet_manager"/>
+    <menuitem id="menu_fleet_reporting_indicative_costs"
+              name="Indicative Costs"
+              parent="menu_fleet_reporting"
+              action="action_fleet_reporting_costs_non_effective"
+              sequence="2"
+              groups="group_fleet_manager"/>
     </data>
 </openerp>
```
Comment (WHPODOO) [inline addons/fleet/views/fleet_views.xml]: Use 
```
<odoo>
      [CODE]
</odoo>
```
no need to use ***<data></data>***

**Comment by mart-e** [comment]
@lla-odoo once you are back with us, please rebase your code, resolve commits, squash the commits into one to move the files and one to migrate the code :)

**Comment by hmo-odoo** [comment]
continue work on https://github.com/odoo/odoo/pull/11949

**Pull request closed without merging**